modules/ROOT/pages/major-changes.adoc

```diff
@@ -9,6 +9,40 @@ This list is in reverse chronological order to keep recent changes at the top.
 
 // To add a new change here, see the template at the end of the file.
 
+== Switch to OCI updates
+
+Starting in Fedora 42, Fedora CoreOS will be updating through OCI images instead of the
+OSTree repo.
+Fedora Change request: https://fedoraproject.org/wiki/Changes/CoreOSOstree2OCIUpdates
+More discussion: https://github.com/coreos/fedora-coreos-tracker/issues/1823
+
+=== Planning
+
+This change has been introduced first in bootimages for the testing stream, for the Fedora 42 rebase. After a few releases
+we will migrate existing machines through a barrier release.
+
+|===
+|Update Stream |Targeted release date
+
+|`next` | 2025-02-04
```

Comment:
Those dates may change so let's write the condition (first release on F42) here instead. We'll update the page once we have the exact releases.